quickstart/buildSrc/src/main/kotlin/ConfigureProfilesTask.kt

```diff
@@ -56,6 +57,24 @@ open class ConfigureProfilesTask : DefaultTask() {
                     option.value = stringValue
                     println("  ${option.envVarName} set to '$stringValue'.\n")
                 }
+
+                OptionType.TEST_MODE -> {
+                    val boolValue = promptForBoolean(option.promptText, default = true)
```

Comment:
@peterkvokacka-da @filmackay-da just want to clarify that the intention here is that the default response in `make setup` for TEST_MODE is `on`. Is this correct?